thirdeye/thirdeye-frontend/app/styles/shared/_styles.scss

```diff
@@ -142,7 +161,7 @@ body {
 .te-label {
   font-weight: normal;
   color:  app-shade(black, 9);
-  margin-bottom: 12px;
+  margin: 15px 0 12px 0;
```

Comment:
BLOCKING COMMENT: this is going to change the styling for a lot of labels in the app. Is this a global change at the art-deco/design level?

Reply:
fixed - wasn't that important.